### DISCUSSION TITLE
How does `custom access permissions > field presets` work?

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15483] [Archive]
I have been fiddling with this for a while and can't get it to apply for create or update, nor have I found another discussion on this, though one likely exists.

Is my object incorrect? If so, what should it be? I'm trying to follow the [create an item](https://docs.directus.io/reference/items.html#create-an-item) object.

<img width="1444" alt="99f11d02-d713-4e10-b0ad-1b65dd9473a8" src="https://user-images.githubusercontent.com/62306630/189203996-cc55d853-4477-4365-bf91-5108e0077968.png">

Additionally, whenever I add in the JSON, it doesn't indicate "custom access permissions"....

<img width="52" alt="Screen Shot 2022-09-08 at 2 10 54 PM" src="https://user-images.githubusercontent.com/62306630/189206344-f4a9a45f-d3f1-4f3d-820d-d097a7d5ae9c.png">

 It just remains a checkmark... which makes sense, as this is a template, not an access permission... but it would be nice to know where template is being used. Should we open a ticket for that to rethink the UI 🤔 

https://user-images.githubusercontent.com/62306630/189205988-d6687bf4-af54-481f-8e81-ba9e41a8fb48.mov

**Comment by rijkvanzanten** [comment]
The structure looks good to me. Presets are effectively a role-level override for default values. What problems where you having? Weren't those values being saved?

**Reply by erondpowell** [reply]
@rijkvanzanten the values were saved within this `config>permissions>presets` menu shown. 
The saved values did not autofill on the item page.

**Reply by erondpowell** [reply]
This is partially working. I do think we do have a bug.

Yesterday, neither the date-published nor the `images` (or any other preset) was being applied as a default in the app for the relevant role.

https://user-images.githubusercontent.com/62306630/189395840-f5cc8fe6-b4c9-4608-8f7b-42023718e81f.mp4
